lib/liveman_web/helpers/ecto_helpers.ex

```diff
@@ -0,0 +1,34 @@
+defmodule LivemanWeb.EctoHelpers do
+  def pretty_errors(errors) do
```

Comment:
I took this code from this [article]( https://medium.com/@brain.amsterdam/prettify-ecto-errors-39512370d006)